types/check.go

```diff
@@ -224,18 +224,20 @@ func (b ByExecuted) Less(i, j int) bool { return b[i].Executed < b[j].Executed }
 func (c *Check) MergeWith(chk *Check) {
```

Comment:
I know it's not directly related to your PR but could you rename `chk` to something else, like `originalCheck`, `oldCheck` or `sourceCheck`? It took few moments to understand which on these two checks was merged into the other and I feel the bug you fixed at lines 227-228 might be related to that complexity (thank you for fixing that btw 👏 )

Reply:
Will do!